packages/@aws-cdk/aws-codepipeline-actions/lib/lambda/invoke-action.ts

```diff
@@ -87,6 +87,13 @@ export class LambdaInvokeAction extends Action {
       resources: [this.props.lambda.functionArn]
     }));
 
+    // allow the Role access to the Bucket, if there are any inputs/outputs
+    if ((this.actionProperties.outputs || []).length > 0) {
+      options.bucket.grantReadWrite(options.role);
```

Comment:
I guess there's no  `grantWrite()`? Should there be?

Reply:
There is actually a `grantWrite` on `IBucket`, but I figured it's not worth the hassle here.

Reply:
Don't get it. Why not:

```ts
if (outputs) { grantWrite(); }
if (inputs) { grantRead(); }
```

Seems simpler and most correct?

Reply:
Fine fine. Will change.

Reply:
Done.